letsencrypt/cli.py

```diff
@@ -328,6 +333,9 @@ def _paths_parser(parser):
     add("--chain-path", default=flag_default("chain_path"),
         help=config_help("chain_path"))
 
+    add("--enroll-autorenew", default=None, action="store_true",
```

Comment:
Dead code. Since `IConfig.entroll_autorenew` is not used anywhere (yet), this shall be removed (and later brought back when implementation is available).